ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -183,12 +187,32 @@ const DATASET_TYPE_PANEL_PROPS = {
   },
 }
 
+DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_VARIANT_MITO] = {
+  ...DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_VARIANT_CALLS],
+  [FREQUENCY_PANEL.name]: {
+    fields: SNP_FREQUENCIES.concat(MITO_FREQUENCIES),
+  },
+  [QUALITY_PANEL.name]: {
+    fields: SNP_QUALITY_FILTER_FIELDS.concat(MITO_QUALITY_FILTER_FIELDS),
+  },
+}
+
+DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_VARIANT_SV] = {
```

Comment:
it is cleaner to keep this in the definition for `DATASET_TYPE_PANEL_PROPS` instead of adding it after like this (so add this in the original `const DATASET_TYPE_PANEL_PROPS = {...}` definition). The only reason to add it after is if you need to reference the existing object the way you do for `DATASET_TYPE_VARIANT_MITO`.